package.json

```diff
@@ -31,23 +31,23 @@
   },
   "dependencies": {
     "@paulcbetts/system-idle-time": "^1.0.4",
-    "electron-updater": "3.0.4",
+    "electron-updater": "4.0.4",
```

Comment:
This should be heavily tested before go into production due to https://github.com/electron-userland/electron-builder/issues/3269